### ISSUE TITLE
URL standard update: opaque paths cannot have trailing space

### ISSUE THREAD

**Issue by annevk** [mdn/content#23320]
In https://github.com/whatwg/url/pull/728 we tweaked the `search` and `hash` setters to account for a minor roundtrip issue. Depending on the level of detail you go into this may impact documentation.

**Comment by annevk** [comment]
The change mentioned above will likely be reverted and we'll instead change the URL parser so that opaque path URLs cannot have their paths end in a non-percent-encoded U+0020. See https://github.com/whatwg/url/pull/844. I suspect that's not the level of detail MDN covers so this can probably be closed, but YMMV.

**Comment by Josh-Cena** [comment]
I'm okay to mention this, especially since we now document escaping rules for different URL components (see for example https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname). It doesn't seem to be widely implemented yet though.

**Comment by annevk** [comment]
Yeah, unfortunately Chrome and Firefox don't prioritize URL parsing interoperability much.

**Comment by Josh-Cena** [comment]
This new percent-encoding behavior is already in Safari TP at least. Other browsers have also implemented the previous behavior which is setter stripping whitespace. So definitely need new content anyway, and of course browser compat.
